pkg/cloud-provider/cloudapi/aws/aws_security.go

```diff
@@ -58,7 +58,7 @@ func buildEc2UserIDGroupPairs(addressGroupIdentifiers []*securitygroup.CloudReso
 }
 
 // buildEc2CloudSgNamesFromRules builds all needed ec2 security group names from address groups in rules and target appliedTo group.
```

Comment:
is this common for AG and AT? is not update comment too

Reply:
It build names from AT and all the AGs in AT rules.